assets/js/blocks/collection-filters/inner-blocks/rating-filter/block.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "woocommerce/collection-rating-filter",
+	"version": "1.0.0",
+	"title": "Collection Rating Filter",
+	"description": "Enable customers to filter the product collection by rating.",
+	"category": "woocommerce",
+	"keywords": [ "WooCommerce" ],
+	"supports": {
+		"interactivity": true
+	},
+	"ancestor": [
+		"woocommerce/collection-filters"
+	],
+	"usesContext": [
+		"collectionData"
+	],
+	"attributes": {
+		"className": {
+			"type": "string",
+			"default": ""
+		},
+		"showCounts": {
+			"type": "boolean",
+			"default": false
+		},
+		"displayStyle": {
+			"type": "string",
+			"default": "dropdown"
+		},
+		"showFilterButton": {
+			"type": "boolean",
+			"default": false
+		},
```

Comment:
Currently we haven't implemented this feature, so I think we shouldn't add it now.